### PR TITLE
New guides for SFA

### PR DESCRIPTION
**Pull request by ihsraham**

### CODE REVIEW

**Comment by yashovardhan** [comment]
Also please update the links and everything inside the guides, it seems to be failing. Most of the live demos have been removed. People can directly just run and check out the projects in their local

File: src/pages/content-hub/guides/sfa-web-google.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
+---
+title: Using Google with Web3Auth SFA Web
```
[the remark this answers is not in the document]
Comment (ihsraham) [reply src/pages/content-hub/guides/sfa-web-google.mdx]: Resolved

File: src/pages/content-hub/guides/sfa-web-google.mdx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+image: "content-hub/guides/banners/google.png"
+description: Learn how to use Web3Auth Single Factor Auth web SDK with Google
```
[the remark this answers is not in the document]
Comment (ihsraham) [reply src/pages/content-hub/guides/sfa-web-google.mdx]: Resolved

File: src/pages/content-hub/guides/sfa-web-google.mdx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+This guide will cover the basics of how to set up your Web3Auth SDK and Google account for the integration and provide you with the links on how to
+develop a basic web application on the blockchain of your choice.
+
+## Quick Start
+
+```bash
+npx degit Web3Auth/web3auth-core-kit-examples/single-factor-auth-web/sfa-web-google-example sfa-web-google-demo && cd sfa-web-google-demo && npm install && npm run start
+```
+
+## How it works?
```
[the remark this answers is not in the document]
Comment (ihsraham) [reply src/pages/content-hub/guides/sfa-web-google.mdx:53]: Added more info.

**Review by yashovardhan — APPROVED** [review]

**Pull request merged**